packages/common/src/Config.ts

```diff
@@ -60,6 +61,17 @@ export interface CCFConfig {
     RESOURCE_TAG_NAMES?: string[]
     CONSUMPTION_CHUNKS_DAYS?: number
   }
+  ALI?: {
+    NAME?: string
+    CURRENT_SERVICES?: { key: string; name: string }[]
+    CURRENT_REGIONS?: string[]
+    INCLUDE_ESTIMATES?: boolean
+    USE_BILLING_DATA?: boolean
```

Comment:
Since support for billing data is the only method implemented, we don't need this config variable. (Only used for toggling between Cloud Usage APIs and Billing Data for usage information)